File: Python/1160.py
t = int(input())

for i in range(0,t):
    pa,pb,ga,gb = map(float, input().split())
    pa = int(pa)
    pb = int(pb)
    ano = 0
    while pb>=pa:
        pa = int(pa*(1 + ga/100))
        pb = int(pb*(1 + gb/100))
        #print('PA {} e o PB {}'.format(pa,pb))
        ano = ano + 1
        if ano >=101:
            print('Mais de 1 seculo.')
            break
    if ano <=100:
        ano = int(ano)
        print('{} anos.'.format(ano))
    

File: CSharp/1160.cs
using System;
using System.Globalization;

class URI
{
    static void Main()
    {
        CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
        int x = int.Parse(Console.ReadLine());

        for (int i = 0; i < x; i++)
        {
            string[] y = Console.ReadLine().Split();
            int pA = int.Parse(y[0]);
            int pB = int.Parse(y[1]);
            decimal tA = decimal.Parse(y[2]) / 100;
            decimal tB = decimal.Parse(y[3]) / 100;
            int anos = 0;

            while (pA <= pB)
            {
                pA += (int)(pA * tA);
                pB += (int)(pB * tB);
                anos++;

                if (anos > 100)
                {
                    break;
                }
            }

            if (anos <= 100)
            {
                Console.WriteLine($"{anos} anos.");
            }
            else
            {
                Console.WriteLine("Mais de 1 seculo.");
            }
        }
    }
}
